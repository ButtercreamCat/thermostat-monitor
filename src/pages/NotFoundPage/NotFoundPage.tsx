import React from "react";
import { Link } from "react-router-dom";
import "./NotFoundPage.scss";

export default function NotFoundPage() {
  return (
    <div className="notFound">
      <h1>Page Not Found</h1>
      <Link to="/">Back to home page</Link>
    </div>
  );
}
