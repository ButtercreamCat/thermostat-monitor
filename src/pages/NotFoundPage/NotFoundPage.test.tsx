import React from "react";
import { shallow } from "enzyme";
import NotFoundPage from "./NotFoundPage";

test("should render NotFoundPage correctly", () => {
  const wrapper = shallow(<NotFoundPage></NotFoundPage>);
  expect(wrapper).toMatchSnapshot();
});
