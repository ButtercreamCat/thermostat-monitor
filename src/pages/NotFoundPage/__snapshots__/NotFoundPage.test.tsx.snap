// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<div
  className="notFound"
>
  <h1>
    Page Not Found
  </h1>
  <Link
    to="/"
  >
    Back to home page
  </Link>
</div>
`;
